AGGREGATE.md

```diff
@@ -98,39 +98,33 @@ at most 32 digits.
 
 ### Attribution trigger registration
 
-Trigger registration will also include two new headers:
-`Attribution-Reporting-Register-Aggregatable-Trigger-Data` is a list of dict
-which generates aggregation keys.
+Trigger registration will also add two new fields to the JSON dictionary of the
+[`Attribution-Reporting-Register-Trigger`
+header](https://github.com/WICG/conversion-measurement-api/blob/main/EVENT.md#triggering-attribution):
 ```jsonc
-[
-// Each dict independently adds pieces to multiple source keys.
-{
-  // Conversion type purchase = 2 at a 9 bit offset, i.e. 2 << 9.
-  // A 9 bit offset is needed because there are 511 possible campaigns, which
-  // will take up 9 bits in the resulting key.
-  "key_piece": "0x400",
-  // Apply this key piece to:
-  "source_keys": ["campaignCounts"]
-},
 {
-  // Purchase category shirts = 21 at a 7 bit offset, i.e. 21 << 7.
-  // A 7 bit offset is needed because there are ~100 regions for the geo key,
-  // which will take up 7 bits of space in the resulting key.
-  "key_piece": "0xA80",
-  // Apply this key piece to:
-  "source_keys": ["geoValue", "nonMatchingKeyIdsAreIgnored"]
-}
-]
-```
-The other new header `Attribution-Reporting-Register-Aggregatable-Values` lists
-an amount of an abstract "value" to contribute to each key, which can be
-integers in [1, 2^16). These are attached to aggregation keys in the order they
-are generated. See the [contribution
-budgeting](#contribution-bounding-and-budgeting) section for more details on how
-to allocate these contribution values.
+  ... // existing fields, such as `event_trigger_data`
 
-```jsonc
-{
+  "aggregatable_trigger_data": [
```

Comment:
nit: Should we just use "event" and "aggregatable" as the top-level keys?

aggregatable_trigger_data is long and a bit redundant with the header name

Reply:
At some point we discussed `event_level_{cfg|cfgs|configs}` and `aggregatable_{cfg|cfgs|configs}`.

Reply:
I like @johnivdel's suggestion in principle, the only problem is that `aggregatable` and `aggregatable_values` living side by side a bit weird. You need some clarification on what the `aggregatable` key is describing that isn't described by `aggregatable_values`.

Reply:
Yeah that's fair. I guess on the aggregate side what we have looks fine.

The event_trigger_data field is a bit ugly as you have trigger_data defined directly inside it but overall I would prefer the names stay aligned.